loafwallet/src/FlowControllers/URLController.swift

```diff
@@ -64,11 +64,6 @@ class URLController : Trackable {
             return true
```

Comment:
Ironically, I think this class (URLController) itself is never used.  But let's be slow and careful.  Just FYI.